getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1483 @@ It will prompt a few options. Select defaults by pressing enter.
+    "isPrivateFile" => false,               // true|false
+    "customCoordinates" => implode(",", ["10", "10", "100", "100"]),    // max: 500 chars
+    "responseFields" => implode(",", ["tags", "customMetadata"]),
+    "extensions" => [       
+        [
+            "name" => "remove-bg",
+            "options" => [  // refer https://docs.imagekit.io/extensions/overview
+                "add_shadow" => true
+            ]
+        ]
+    ],
+    "webhookUrl" => "https://example.com/webhook",
+    "overwriteFile" => true,        // in case of false useUniqueFileName should be true
+    "overwriteAITags" => true,      // set to false in order to preserve overwriteAITags
+    "overwriteTags" => true,
+    "overwriteCustomMetadata" => true,
+    // "customMetadata" => [
+    //         "SKU" => "VS882HJ2JD",
+    //         "price" => 599.99,
+    // ]
+];
+
+// Attempt File Uplaod
+$uploadFile = $imageKit->upload([
+    'file' => 'your_file',                  //  required, "binary","base64" or "file url"
+    'fileName' => 'your_file_name.jpg',     //  required
+    'options' => $uploadOptions             // optional
+]);
+```  
 
-You can manage [security settings](../../features/security/#restricting-unsigned-urls) from the dashboard to prevent unsigned URLs usage. In that case, if the URL doesn't have signature `ik-s` parameter or the signature is invalid, ImageKit will return a forbidden error instead of an actual image.
 
-## Server-side file uploading
+## ImageKit Media API
 
-The SDK provides a simple interface using the `$imageKit->upload()` method to upload files to the ImageKit Media Library. It accepts all the parameters supported by the [ImageKit Upload API](../../api-reference/upload-file-api/server-side-file-upload.md).
+The SDK provides a simple interface for all the following [Media APIs](https://docs.imagekit.io/api-reference/media-api) to manage your files.
 
-The `uploadFiles()` method requires at least the `file` and the `fileName` parameter to upload a file and returns a JSON response. You can pass other parameters supported by the ImageKit upload API using the same parameter name as specified in the upload API documentation. For example, to specify tags for a file at the time of upload use the `tags` parameter as specified in the [documentation here](../../api-reference/upload-file-api/server-side-file-upload.md).
+### 1. List and Search Files
 
-```javascript
-$imageKit->upload(array(
-    "file" => "your_file", // required, can be base64 or binary or a remote url
-    "fileName" => "your_file_name.jpg", // required
-    ... // Additional Parameters can be refered to Server Side file Upload Documentation, https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload
-));
+This API can list all the uploaded files and folders in your [ImageKit.io](https://docs.imagekit.io/api-reference/media-api) media library.
+
+#### Example
+```php
+$listFiles = $imageKit->listFiles();
+```
+#### Response
+```json
```

Comment:
API responses is well documented in docs.imagekit.io and we should add links of it everywhere instead of repeating.